### PR TITLE
Refactor: Redefine group class wrapper, add type param

### PR DESCRIPTION
**Pull request by SangTran-127**
## Type

- [ ] Feature
- [x] Bug fix
- [ ] CI/CD
- [ ] Documentation
- [ ] Other

## Description

Brief description of the changes made.

### CODE REVIEW

File: packages/serverless/src/domain/use-case/group.ts [hunk 1]
```diff
@@ -1,254 +1,350 @@
...
+type TParamGroup = TGroup & {
+  databaseName: string;
+};
 
-  const group = await modelGroup.findGroup(groupName, session);
+type TParamGroupExist = Pick<TParamGroup, 'databaseName' | 'groupName'>;
 
-  return group != null;
-}
+type TParamGroupCreate = TParamGroup;
```
Comment (chiro-hiro) [inline packages/serverless/src/domain/use-case/group.ts:16]: I created a new file it named `param.ts`, for common param.

File: packages/serverless/src/domain/use-case/group.ts [hunk 2]
```diff
@@ -1,254 +1,350 @@
...
-        },
-        { $set: { groupName: newGroupName } }
-      );
-      return result && result.modifiedCount === 1;
+      const group = await modelGroup.findOne({ groupName }, { session });
+
+      if (group) {
+        const modelUser = new ModelUser();
+        // Check list of user exist
+        if (await modelUser.areUsersExist(listUserName)) {
```
Comment (chiro-hiro) [inline packages/serverless/src/domain/use-case/group.ts:330]: Must be `areListUserExist`, no plural please.

Comment (SangTran-127) [reply packages/serverless/src/domain/use-case/group.ts:330]: Let keep it in this PR for only group. I do the refactor for User in next PR

File: packages/serverless/src/domain/use-case/group.ts (hunk 1 above)
Comment (SangTran-127) [reply packages/serverless/src/domain/use-case/group.ts]: Solved, move to param from common package

**Review by chiro-hiro — APPROVED** [review]

**Pull request merged**